test/product-import.spec.coffee

```diff
@@ -510,6 +510,19 @@ describe 'ProductImport unit tests', ->
         done()
       .catch done
 
+    it 'should throw error in case of too big query', (done) ->
+      exptectedErrorMessage = 'product fetch query size: 10952 bytes, exceeded the supported size, please try with a smaller batch size.'
```

Comment:
@butenkor  This is not a permanent solution. It is a temporary solution till we figure out a way to solve this ticket: https://github.com/sphereio/sphere-product-import/issues/5